Library/Styles/Colors.swift

```diff
@@ -98,9 +98,9 @@ extension UIColor {
     return .hex(0x656868)
   }
 
-  /// 0x3B5998
+  /// 0x1877F2
```

Comment:
Since this is a generated file we will have to update the actual source `Colors.json` otherwise this will be removed next time we generate colors

Reply:
Cheers. I set up time with @cdolm92 to pair on closing-out this PR, so she'll help me with this.

Reply:
Resolving as Christella helped me do it properly. I removed my hardcoded changes to the Colors.swift file—instead doing them in the Colors.json, and then running the script.